backend/src/main/resources/doc/openapi.json

```diff
@@ -1107,6 +1105,87 @@
                         }
                     }
                 }
+            },
+            "ExaminationStatus": {
+                "type": "string",
+                "title": "ExaminationStatus",
+                "description": "Status enum of the examination",
+                "enum": [
+                    "NEW",
+                    "TO_BE_CONFIRMED",
+                    "CONFIRMED",
+                    "CANCELED",
+                    "UNKNOWN"
+                ]
+            },
+            "PreventionStatus": {
+                "type": "object",
+                "title": "PreventionStatus",
+                "description": "Overview status of examinations",
+                "properties": {
+                    "id": {
+                        "type": "integer",
+                        "format": "int64"
+                    },
+                    "examinationType": {
+                        "$ref": "#/components/schemas/ExaminationTypeEnum"
+                    },
+                    "intervalYears": {
+                        "type": "integer"
+                    },
+                    "date": {
+                        "type": "string",
+                        "format": "date-time"
+                    },
+                    "firstExam": {
+                        "type": "boolean"
+                    },
+                    "priority": {
+                        "type": "integer"
+                    },
+                    "state": {
+                        "$ref": "#/components/schemas/ExaminationStatus"
+                    },
+                    "count": {
+                        "type": "integer"
+                    }
+                },
+                "required": [
+                    "examinationType",
+                    "intervalYears",
+                    "firstExam",
+                    "priority",
+                    "state",
+                    "count"
+                ]
+            },
+            "ExaminationRecord": {
+                "title": "ExaminationRecord",
+                "type": "object",
+                "description": "Examination object to update or create an object",
+                "properties": {
+                    "id": {
```

Comment:
V Account posilame v API uuid (i kdyz je ID based), coz je rozumne a neleakujeme tim "pocet uzivatelu". Nechceme se toho principu drzet a posilat uuid misto ID vsude?

Reply:
uuid u uzivatele je ID generovane ve Firebase. Muzeme zavest dalsi synteticke UUID i u prohlidky. To bychom si museli generovat sami.

Reply:
Upraveno, prislo mi to jako dobry napad.